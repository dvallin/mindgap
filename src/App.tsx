import * as React from 'react'
import { Route } from 'react-router-dom'

import RecipePage from './recipes/RecipePage'
import BatchPage from './batches/BatchPage'

export default (): JSX.Element => (
  <div className="container">
    <Route exact path="/recipes/:id?" component={RecipePage} />
    <Route exact path="/batches/:id?" component={BatchPage} />
  </div>
)
