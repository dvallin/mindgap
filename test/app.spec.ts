import App from '../src/app'
import { createShallowMount } from './test-utils'

describe('App', () => {
  const mount = createShallowMount(App, {})
  it('renders', () => {
    const component = mount()
    expect(component).toMatchSnapshot()
  })
})
