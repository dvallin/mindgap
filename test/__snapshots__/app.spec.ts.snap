// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  id="app"
>
  <Provider
    initialState={[Function]}
    onChange={[Function]}
  >
    <Component />
    <u>
      <Component
        path="/batches/:id"
      />
      <Component
        path="/batches"
      />
      <Component
        path="/recipes/:id"
      />
      <Component
        path="/recipes"
      />
      <Component
        path="/"
      />
      <Component
        default={true}
      />
    </u>
  </Provider>
</div>
`;
